airflow/www/utils.py

```diff
@@ -448,7 +448,42 @@ def __call__(self, field, **kwargs):
         return wtforms.widgets.core.HTMLString(html)
 
 
-class UtcFilterConverter(FilterConverter):
+class BaseUtcDateTimeFilter(object):
```

Comment:
@ashb Aren't you supposed to derive from `BaseDateTimeFilter` instead of `object` ?

Reply:
`BaseDateTimeFilter` comes in from `sqlafilters.DateTimeEqualFilter` etc - this is more of a "mix-in" than it's.....

Welll _only_ the `>=` filter works. Thanks for catching